src/view/pane/chat.js

```diff
@@ -550,10 +550,11 @@ Candy.View.Pane = (function(self, $) {
        * Show the login form modal
        *
        * Parameters:
-       *  (String) message - optional message to display above the form
+       *  (String) messageKey - optional message to display above the form
        *  (String) presetJid - optional user jid. if set, the user will only be prompted for password.
        */
-      showLoginForm: function(message, presetJid) {
+      showLoginForm: function(messageKey, presetJid) {
+        var message = $.i18n._(messageKey);
```

Comment:
I'm sure I'm not alone when I say that we appreciate your drive to contribute to this project. It's clear we all care about this project. However, that also means that we want to make sure that the code that goes in here is as clean and neat as possible. We're not necessarily getting paid either. I'm not sure anything unpleasant happened here other than you became defensive of your code that you wrote and weren't open/able to receive some friendly constructive criticism. 

I hope you can consider this a growing experience and that you contribute to Candy and/or OSS in the future. :+1: